lib/presentation/widget/shared_space/shared_space_viewmodel.dart

```diff
@@ -224,6 +224,14 @@ class SharedSpaceViewModel extends BaseViewModel {
     store.dispatch(SharedSpaceClearSelectedSharedSpacesAction());
   }
 
+  void goToSharedSpaceDetails(SharedSpaceNodeNested sharedSpaceNodeNested) {
+    _appNavigation.popBack();
+    _appNavigation.push(
+      RoutePaths.sharedSpaceDetails,
+      arguments: sharedSpaceNodeNested.sharedSpaceId
```

Comment:
you should put your data inside a wrapper class like `SharedSpaceDetailArguments`. So other people will easily know where they can put arguments and where they can get it without a doubt

Reply:
hmm let make this more specific, can you create an `abstract class RouteArgument`, and the `push()` inside `AppNavigation` has the param `arguments` with type of `RouteArgument`. WDYT?

Reply:
Sure great idea, I just just dont want to put many informations in SharedSpaceDetailsArguments so we don't rely on it too much